src/app/components/list/view-switcher/index.html

```diff
@@ -4,7 +4,7 @@
 
   <sky-demo-page-summary>
     <p>
-      The list view switcher lets the user switch between different views of the same data.
+      The list view switcher module displays toolbar options that enable users to switch between different views of the same data.
     </p>
   </sky-demo-page-summary>
 
```

Comment:
@Blackbaud-AlexKingman, are you planning to create the demo for this component? Or should I look into trying to set that up? (I assume it'll be much quicker for a developer to create the demo, but I can work on it if necessary.)

Reply:
I can work on creating a demo.

Reply:
Resolving this issue as we've decided to move forward with the docs minus the demo and circle back on the demo and code samples.